voice/bxml/callbacks/disconnect.md

```diff
@@ -1,6 +1,19 @@
 {% method %}
 ## Disconnect Event
-The Disconnect event is fired when a call ends, for any reason.
+
+The Disconnect event is fired when a call ends, for any reason. The `cause` for a disconnect event on a call can be:
+- `hangup`: there was no more BXML to execute in it; it indicates that the call ended normally.
+- `busy`: the callee was busy.
+- `timeout`: the call wasn't answered before the `callTimeout` was reached.
+- `cancel`: while the call was ringing, it was cancelled by its originator.
+- `rejected`: the call was rejected by the callee.
+- `callback-error`: a BXML callback couldn't be delivered.
+- `invalid-bxml`: an invalid bxml was tried to be played in the call.
+- `application-error`: some non-supported action was tried on the call, e.g. trying to play a .ogg audio.
```

Comment:
```suggestion
- `application-error`: some unsupported action was tried on the call, e.g. trying to play a .ogg audio.
```
Not sure I like "some" instead of "a"/"an" either, but I'll leave that for someone better at English.